man/systemd.exec.xml

```diff
@@ -1047,8 +1047,11 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         the <varname>JoinsNamespaceOf=</varname> directive, see
         <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
         details. Note that this option will disconnect all socket families from the host, this includes AF_NETLINK and
-        AF_UNIX.  The latter has the effect that AF_UNIX sockets in the abstract socket namespace will become
-        unavailable to the processes (however, those located in the file system will continue to be accessible).</para>
+        AF_UNIX.  The former means that device configuration events (as sent by
+        <citerefentry><refentrytitle>systemd-udevd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>) are
+        not delivered by the unit's processes. The latter has the effect that AF_UNIX sockets in the abstract socket
```

Comment:
I think wording could be improved here, I'm reading this and it still doesn't make sense to me.
* "sent by systemd-udevd.service" -> Isn't udevd receiving events through the netlink?
* "delivered by the unit's processes" -> which unit?

Also, perhaps being explicit about AF_NETLINK and AF_UNIX would make this easier to scan than when using "former" or "latter" which requires a lookbehind.

Reply:
> Isn't udevd receiving events through the netlink?

Yes, and it augments it with metadata from udev rules and then sends them to its clients, again via AF_NETLINK. So it recvs *and* receives them via AF_NETLINK. From an app perspective the messages originate from udev however, and that's what we should focus on

Reply:
> "delivered by the unit's processes" -> which unit?

uh, this is supposed to say "delivered to", not "delivered by"...

And it's the unit that has PrivateNetwork= set that is meant here, which I think is clear enough, no?

Reply:
Perhaps using something like "received from" will improve it? Like "device configuration events *received from* systemd-udevd.service"... Otherwise, looks good.